takwimu/templates/takwimu/_includes/navbar/_dropdown.html

```diff
@@ -11,12 +11,17 @@
   <div class="container">
     <div class="row">
       <div class="col-3">
-        <h2 class="font-weight-bold">{% block dropdown_title %}Some title{% endblock %}</h2>
-        <p class="font-weight-light">{% block dropdown_desc %}Lorem ipsum{% endblock %}</p>
+        <h2 class="font-weight-bold text-right pr-3">{% block dropdown_title %}Some title{% endblock %}</h2>
+        <p class="font-weight-light text-right pr-3" style="line-height:1.61;font-weight:100;">{% block dropdown_desc %}Lorem ipsum{% endblock %}</p>
       </div>
       <div class="col-9 border-left d-flex align-items-center" style="border-left-style: dashed !important;">
         {% block dropdown_content %}{% endblock %}
       </div>
     </div>
   </div>
 </div>
+<style>
+  ul li u {
+    color:#7ab1f2;
+  }
+</style>
```

Comment:
Fix requests:
1. Move where this is defined. Here it shall be generated 4 times or whatever number of dropdowns we have.
1. Add a CSS selector that limits style application to the dropdown.